functions/concepts/requests/index.js

```diff
@@ -16,16 +16,18 @@
 
 // [START functions_concepts_requests]
 const fetch = require('node-fetch');
+const functions = require('@google-cloud/functions-framework');
 
 /**
  * HTTP Cloud Function that makes an HTTP request
  *
  * @param {Object} req Cloud Function request context.
  * @param {Object} res Cloud Function response context.
  */
-exports.makeRequest = async (req, res) => {
+functions.http('makeRequest', (req, res)  => {
   const url = 'https://example.com'; // URL to send the request to
-  const externalRes = await fetch(url);
-  res.sendStatus(externalRes.ok ? 200 : 500);
-};
+  fetch(url).then((externalRes) => {
```

Comment:
Can we revert this change? `await` is considered cleaner than `.then`.

Reply:
Done